app/views/admin/shared/_sidenav_billing.html.haml

```diff
@@ -11,8 +11,9 @@
     = tab t("facilities.disputed_orders.head"), facility_disputed_orders_path, (sidenav_tab == "disputed_orders")
   %li.divider
   = tab t('facility_accounts.accounts_receivable.head'), facility_accounts_receivable_path, (sidenav_tab == 'accounts_receivable')
-  %li.divider
-  = tab Account.model_name.human.pluralize, facility_accounts_path, (sidenav_tab == 'accounts')
+  - if current_user.billing_administrator?
```

Comment:
Could we add a condition the the accounts controllers to switch which sidebar displays based on whether you're in billing context or users context?